numba/tests/test_dyn_array.py

```diff
@@ -1757,15 +1760,24 @@ def test_vstack(self):
         pyfunc = np_vstack
         cfunc = nrtjit(pyfunc)
 
+        # Since np.row_stack is an alias for np.vstack, it does not need a
+        # separate Numba implementation. For every test for np.vstack, the same
+        # test for np.row_stack has been added.
+        pyfunc_rowstack = np_row_stack
+        cfunc_rowstack = nrtjit(pyfunc_rowstack)
```

Comment:
Create a for loop instead:

```suggestion
        for pyfunc in (np_vstack, np_row_stack):
            cfunc = nrtjit(pyfunc)
            ...
```